### ISSUE TITLE
README out of date

### ISSUE THREAD

**Issue by martindale** [matrix-org/matrix-appservice-slack#86]
Seems at least one parameter is missing in the `README`:
```
{"instanceContext":"#","resolutionScope":"anon-schema://2e73ff33a15403aa43f9e5cf5ac9b7ddb97e7a5e/#","constraintName":"required","constraintValue":["homeserver","slack_hook_port","inbound_uri_prefix","bot_username","username_prefix"],"desc":"missing: inbound_uri_prefix","kind":"ObjectValidationError"}
```
Looks generally out of date; would be nice to have a maintainer review and update.

**Comment by barudaret** [comment]
Same.  The missing parameter is inbound_uri_prefix btw.  I think this may have something to do with why I can't get my slack bridge to create it's own slackbot

**Comment by barudaret** [comment]
Same. The missing parameter is inbound_uri_prefix btw

**Comment by Half-Shot** [comment]
I believe this is fixed in #202
